pkg/obfuscate/sql_tokenizer.go

```diff
@@ -634,7 +634,7 @@ func (tkn *SQLTokenizer) scanNumber(seenDecimalPoint bool) (TokenKind, []byte) {
 			}
 			// octal int
 			if seenDecimalDigit {
-				// tkn.setErr called in caller
+				tkn.setErr("invalid number")
```

Comment:
Can we make a more specific error message? Why is it an invalid number? If we end up debugging this, we'll have an easier time knowing where the error occurred vs. having two places with the same error message.

Reply:
Done.